src/ostream

```diff
@@ -416,6 +439,24 @@ namespace std {
 		return out;
 	}
 
+#ifdef ARDUINO_ARCH_AVR
+	//support strings from flash memory [PI]
+	//stolen from Arduino Print.cpp
+	//sample use: cout << F("Hello") << endl;
+	template<class traits> _UCXXEXPORT basic_ostream<char,traits>&
+		operator<<(basic_ostream<char,traits>& out, const __FlashStringHelper* c) //[PI: New] stolen from Arduino Print.cpp
```

Comment:
Is this specialization really needed? It looks like the implementation is identical to the generate `charT` version above, so it can probably be omitted?

Reply:
I mean that the implementation of this specialization: https://github.com/mike-matera/ArduinoSTL/blob/f42634497072bee700f2c8b4f48141af9ce15649/src/ostream#L442-L458

Is identical to this specialization: https://github.com/mike-matera/ArduinoSTL/blob/f42634497072bee700f2c8b4f48141af9ce15649/src/ostream#L415-L431

So I think the first one is redundant and should be caught by the second one (with `charT = char`)?

Reply:
May be Mike just copied some of my code...
Should test again if Mikes code is working in my case.